src/components/ListOfTipsAndComments.vue

```diff
@@ -137,8 +136,11 @@ export default {
   mounted() {
     this.reloadData();
 
-    EventBus.$on('reloadData', () => {
-      this.reloadData();
+    this.$watch(() => this.reloading, (reload) => {
```

Comment:
looks like overhead to me, whats the benefit of this change?

Reply:
If I understood correctly the removal of the event bus.

Reply:
the benefit of removing the event bus is removing the event bus, whats the benefit of this change actually?